src/Symfony/Component/Cache/Traits/GetTrait.php

```diff
@@ -11,48 +11,62 @@
 
 namespace Symfony\Component\Cache\Traits;
 
-use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Cache\Adapter\AdapterInterface;
 use Symfony\Component\Cache\CacheItem;
 use Symfony\Component\Cache\Exception\InvalidArgumentException;
 use Symfony\Component\Cache\LockRegistry;
+use Symfony\Contracts\Cache\CacheInterface;
+use Symfony\Contracts\Cache\ItemInterface;
 
 /**
- * An implementation for CacheInterface that provides stampede protection via probabilistic early expiration.
- *
- * @see https://en.wikipedia.org/wiki/Cache_stampede
- *
```

Comment:
I think we should move the comment where you moved the implementation.

Reply:
The comment is on the contracts interface now, that's why I removed it here.